tests/test_camvid_format.py

```diff
@@ -0,0 +1,196 @@
+from functools import partial
+import numpy as np
+import os.path as osp
+from collections import OrderedDict
+
+from unittest import TestCase
+
+from datumaro.components.extractor import (Extractor, DatasetItem,
+    AnnotationType, Mask,
+)
+from datumaro.components.project import Project
+from datumaro.util.test_utils import (TestDir, compare_datasets,
+    test_save_and_load)
+import datumaro.plugins.camvid_format as Camvid
+from datumaro.plugins.camvid_format import CamvidImporter, CamvidConverter
+
+class CamvidFormatTest(TestCase):
+    def test_can_write_and_parse_labelmap(self):
+        src_label_map = Camvid.CamvidLabelMap
+
+        with TestDir() as test_dir:
+            file_path = osp.join(test_dir, 'label_colors.txt')
+            Camvid.write_label_map(file_path, src_label_map)
+            dst_label_map = Camvid.parse_label_map(file_path)
+
+            self.assertEqual(src_label_map, dst_label_map)
+
+DUMMY_DATASET_DIR = osp.join(osp.dirname(__file__), 'assets', 'camvid_dataset')
+
+class TestExtractorBase(Extractor):
+    def _label(self, camvid_label):
+        return self.categories()[AnnotationType.label].find(camvid_label)[0]
+
+    def categories(self):
+        return Camvid.make_camvid_categories()
+
+class CamvidImportTest(TestCase):
+    def test_can_import(self):
+        class DstExtractor(TestExtractorBase):
+            def __iter__(self):
+                return iter([
+                    DatasetItem(id='0001TP_008550', subset='test',
+                        image=np.ones((2, 5, 3)),
+                        annotations=[
+                            Mask(image=np.array([[1, 1, 1, 0, 0], [0, 0, 0, 0, 0]]), label=0),
```

Comment:
Let's split the array definition into few lines to simplify reading. You can also simplify the test by leaving only one line in the image.